addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -343,8 +344,23 @@ class EnvironmentScService extends Service {
       fetchCidr
     ) {
       const { currentIngressRules } = await this.getSecurityGroupDetails(requestContext, env);
+
+      // Validate the CFT output if RStudio is exist then retrieve the describe rule and inject into the currentIngressRules
+      const { MetaConnection1Type, ListenerRuleARN } = cfnOutputsArrayToObject(env.outputs);
+      if (MetaConnection1Type === 'RStudioV2') {
+        const albService = await this.service('albService');
+        const resolvedVars = { ruleARN: ListenerRuleARN, projectId: env.projectId };
+        const ruleSourceIps = await albService.describeRules(requestContext, resolvedVars);
+        currentIngressRules.map(obj => {
+          if (obj.fromPort === 443) {
+            obj.cidrBlocks = ruleSourceIps;
+          }
+          return obj;
+        });
+      }
```

Comment:
Can we add a unit test for this branch?

Reply:
Separate method has been created for the describeELBRule and incorporated the test cases as well. Please review